agent/README.md

```diff
@@ -87,6 +88,20 @@ Note that defining `-Djava.io.tmpdir` will change the temp directory that is bei
 If there is no log file at that location, it means the agent didn't even start and you have not configured it correctly.
 Check your applications console output for error messages.
 
+#### Teamscale-Specific Proxy Settings
+It is possible to specify teamscale-specific proxy settings that take prevalence over the system properties of the JVM.
+If no or not all teamscale-specific proxy settings are provided the agent will fall back to the system properties of the
+JVM.
+
+It is also possible to specify these options by prefixing the JVM flags for proxies with `teamscale.`,
+for example `-Dteamscale.https.proxyHost`.
+
+The following options are available:
+- `proxy-host`: The host name of the proxy server.
+- `proxy-port`: The port of the proxy server.
+- `proxy-user`: The username for the proxy server.
+- `proxy-password`: The password for the proxy user.
```

Comment:
I adressed this in my other commend. The option names do not match what we check for.